setup.py

```diff
@@ -235,11 +241,6 @@ def initialize_options(self) -> None:
     mysql_dependencies
 )
 
-gcs_dependencies = [
-    'google-cloud-storage'
-] + smart_open_dependencies
-
-
```

Comment:
The rest of the changes here are just re-topo-sorting the variable definitions so that we only use things after they are defined.